src/main/java/com/example/sinitto/guardGuideline/service/GuardGuidelineService.java

```diff
@@ -0,0 +1,75 @@
+package com.example.sinitto.guardGuideline.service;
+
+
+import com.example.sinitto.guard.exception.SeniorNotFoundException;
+import com.example.sinitto.guard.repository.SeniorRepository;
+import com.example.sinitto.guardGuideline.dto.GuardGuidelineRequest;
+import com.example.sinitto.guardGuideline.dto.GuardGuidelineResponse;
+import com.example.sinitto.guardGuideline.entity.GuardGuideline;
+import com.example.sinitto.guardGuideline.entity.GuardGuideline.Type;
+import com.example.sinitto.guardGuideline.exception.GuardGuidelineNotFoundException;
+import com.example.sinitto.guardGuideline.repository.GuardGuidelineRepository;
+import com.example.sinitto.member.entity.Senior;
+import com.example.sinitto.member.repository.MemberRepository;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.List;
+
+@Service
+public class GuardGuidelineService {
+
+    private final GuardGuidelineRepository guardGuidelineRepository;
+    private final SeniorRepository seniorRepository;
+
+    public GuardGuidelineService (GuardGuidelineRepository guardGuidelineRepository, SeniorRepository seniorRepository, MemberRepository memberRepository){
```

Comment:
```suggestion
    public GuardGuidelineService (GuardGuidelineRepository guardGuidelineRepository, SeniorRepository seniorRepository){
```
여기 안쓰이는게 매게변수로 있네요. 제거해주시면 좋겠습니다!

Reply:
넵 확인했습니다!